server/game/cards/06-WoE/SpaceInvaders.js

```diff
@@ -0,0 +1,37 @@
+const Card = require('../../Card.js');
+
+class SpaceInvaders extends Card {
+    // Play: Reveal any number of creatures from your hand. Make each
+    // creature revealed this way a token creature as if the card was
+    // on top of your deck.
+    setupCardAbilities(ability) {
+        this.play({
+            target: {
+                activePromptTitle: 'Choose which cards to reveal',
+                mode: 'unlimited',
+                controller: 'self',
+                location: 'hand',
+                cardType: 'creature',
+                gameAction: ability.actions.makeTokenCreature((context) => ({
+                    target: context.player,
+                    amount: context.target.length,
+                    cards: context.target,
+                    cardLocation: 'hand'
+                }))
+            },
+            effect: 'reveal and tokenize {1}{2} {0}',
```

Comment:
is the {0} needed at the end or is it a left over?

Reply:
The `{0}` is the list of creatures.

Reply:
Understood